website/content/en/docs/advanced-topics/scorecard/custom-tests.md

```diff
@@ -286,15 +286,16 @@ check Kube resources within your tests, or even create custom resources. Your
 custom test image is being executed within a Pod, so you can use an in-cluster
 connection to invoke the Kube API.
 
+<!-- TODO: this file shouldn't refer to the top-level operator-sdk repo as a reference, but a sample (in testdata?) -->
 
 [client_go]: https://github.com/kubernetes/client-go
-[olm_tests]: https://github.com/operator-framework/operator-sdk/blob/master/internal/scorecard/tests/olm.go
-[basic_tests]: https://github.com/operator-framework/operator-sdk/blob/master/internal/scorecard/tests/basic.go
-[config_yaml]: https://github.com/operator-framework/operator-sdk/blob/master/internal/scorecard/testdata/bundle/tests/scorecard/config.yaml
-[scorecard_main_func]: https://github.com/operator-framework/operator-sdk/blob/master/images/scorecard-test/cmd/test/main.go
+[olm_tests]: https://github.com/operator-framework/operator-sdk/blob/09c3aa14625965af9f22f513cd5c891471dbded2/internal/scorecard/tests/olm.go
+[basic_tests]: https://github.com/operator-framework/operator-sdk/blob/09c3aa14625965af9f22f513cd5c891471dbded2/internal/scorecard/tests/basic.go
+[config_yaml]: https://github.com/operator-framework/operator-sdk/blob/09c3aa14625965af9f22f513cd5c891471dbded2/internal/scorecard/testdata/bundle/tests/scorecard/config.yaml
+[scorecard_main_func]: https://github.com/operator-framework/operator-sdk/blob/09c3aa14625965af9f22f513cd5c891471dbded2/images/scorecard-test/main.go
 [custom_scorecard_repo]: https://github.com/operator-framework/operator-sdk
 [user_doc]: /docs/advanced-topics/scorecard/scorecard/
-[scorecard_binary]: https://github.com/operator-framework/operator-sdk/blob/master/images/custom-scorecard-tests/cmd/test/main.go
-[sample_makefile]: https://github.com/operator-framework/operator-sdk/blob/master/Makefile
+[scorecard_binary]: https://github.com/operator-framework/operator-sdk/blob/09c3aa14625965af9f22f513cd5c891471dbded2/images/custom-scorecard-tests/main.go
+[sample_makefile]: https://github.com/operator-framework/operator-sdk/blob/09c3aa14625965af9f22f513cd5c891471dbded2/Makefile
 [kustomize-patchJson6902]: https://kubernetes-sigs.github.io/kustomize/api-reference/kustomization/patchesjson6902/
```

Comment:
I think that we need to find a better way for the docs links. 
IMO master should point out to the master branch when the tag branch should use the tag one. 
So, we might need to make it be dynamic and programmatic as well or some step in the release process. 

wdyt @asmacdo @joelanford @jmrodri

Reply:
curious why we would link to sha's instead of tags as well?   Is this a new docs policy to follow?

Reply:
If the code change then, we should no longer link to the old one in the master, however, the CI will pass because we are using the commit. Am I right? Is that the behaviour that we are looking for?